src/components/calendar/Calendar.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import FullCalendar from "@fullcalendar/react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import timeGridPlugin from "@fullcalendar/timegrid";
+import interactionPlugin from "@fullcalendar/interaction"; // needed for dayClick
+
+import "./main.scss";
+
+export default class Calendar extends React.Component {
+  calendarComponentRef = React.createRef();
+  state = {
+    calendarWeekends: true,
+    // instead of having calendar events in state, simply pass them in on props
+    calendarEvents: [
```

Comment:
"calendarEvents" will be an array passed in on props once we get to it. This is just so we can see the component work.